src/utils.rs

```diff
@@ -234,9 +233,10 @@ pub fn felt252_str(value: &str) -> Felt {
     let value = value
         .parse::<BigInt>()
         .expect("value must be a digit number");
-    let value = match value.sign() {
-        Sign::Minus => &*PRIME - value.neg().to_biguint().unwrap(),
-        _ => value.to_biguint().unwrap(),
+
+    let value = match value.to_biguint() {
+        Some(value) => value,
+        None => &*PRIME - value.neg().to_biguint().expect("is always positive"),
```

Comment:
This unwrap can be removed altogether. Check out [this snippet](https://github.com/lambdaclass/cairo_native/blob/main/src/libfuncs/felt252.rs#L328-L333) for more info.

Reply:
Aah, I couldn't figure out how to remove that unwrap. Thanks. This would also work, right?:
```rust
    let value = match value.sign() {
        Sign::Minus => &*PRIME - value.magnitude(),
        _ => value.magnitude().clone(),
    };
```